packages/design-system/src/components/OcCheckbox/OcCheckbox.vue

```diff
@@ -5,8 +5,13 @@
       v-model="model"
       type="checkbox"
       name="checkbox"
-      :class="classes"
-      class="m-0.5 border-2 border-role-outline outline-0 focus-visible:outline outline-role-secondary"
+      class="oc-checkbox m-0.5 border-2 border-role-outline outline-0 focus-visible:outline outline-role-secondary rounded-sm checked:bg-white disabled:bg-role-surface-container-low indeterminate:bg-white bg-transparent inline-block overflow-hidden"
+      :class="{
+        'oc-checkbox-checked bg-white': isChecked,
```

Comment:
is it doing the same that we have `bg-white` when `isChecked` is true and `checked:bg-white` in the static class list?

Reply:
The last time I checked/tried, `isChecked` and the native `:checked` behaved a bit different for some reason, so I kept both 🤷 Didn't dive too deep into the implementation though.